src/cli/dcos_docker/commands/destroy_loopback_sidecar.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+import click
+import click_spinner
+import docker
+
+from dcos_e2e.docker_utils import DockerLoopbackVolume
+
+from ._common import (
+    NODE_TYPE_LABEL_KEY,
+    NODE_TYPE_LOOPBACK_SIDECAR_LABEL_VALUE,
+    SIDECAR_NAME_LABEL_KEY,
+    docker_client,
+)
+
+
+def _loopback_sidecar_by_name(
+    name: str,
+) -> Optional[docker.models.containers.Container]:
+    client = docker_client()
+    filters = {
+        'label': [
+            '${key}=${value}'.format(
+                key=NODE_TYPE_LABEL_KEY,
+                value=NODE_TYPE_LOOPBACK_SIDECAR_LABEL_VALUE,
+            ),
+            '${key}=${value}'.format(
+                key=SIDECAR_NAME_LABEL_KEY,
+                value=name,
+            ),
+        ],
+    }
+    containers = client.containers.list(filters=filters)
+    assert len(containers) <= 1
```

Comment:
Will it be clear to a user what the problem is if this `AssertionError` is raised?

Reply:
I'll add an error message here.